editor/js/Loader.js

```diff
@@ -270,7 +270,7 @@ function Loader( editor ) {
 						scene.name = filename;
 
 						editor.addAnimation( scene, result.animations );
-						editor.execute( new AddObjectCommand( editor, scene ) );
+						editor.execute( new SetSceneCommand( editor, scene ) );
```

Comment:
The variable is called `scene`, but it is actually of type `THREE.Group`. So using `AddObjectCommand` is actually fine.